e2e/pipeline/pipeline_test.go

```diff
@@ -87,10 +87,19 @@ var _ = Describe("pipeline flow", func() {
 			})
 			Expect(err).NotTo(HaveOccurred())
 		})
-		It("env ls should list both envs", func() {
+		It("qa env init should succeed", func() {
```

Comment:
I wonder if the `qa` env is necessary because the less env we have the less run time it will be. Also do you know how long does it usually take after this change?

Reply:
PH's comment makes me think that it's probably a good idea to create two e2e tests for pipelines. What do you think?

If we go by the idea that each e2e test should test common scenarios, then we see two common scenarios for pipelines:
- Either users have a trunk pipeline `src -> test -> prod`
- Or users have separate pipelines for each env
   -  `src -> test`
   - `src -> prod`

Because of it maybe we should split the test?

Reply:
I am making a separate multi-pipeline e2e test, and restoring the former (single) pipeline test. But this will make the e2e tests take longer as a whole, because we'll be creating and deleting the `test` and `prod` envs twice.  Does this concern you, @iamhopaul123?

Reply:
No worries! I think making it a separate one sounds good to me.